purchase_discount/models/purchase_order.py

```diff
@@ -13,18 +13,14 @@ class PurchaseOrderLine(models.Model):
 
     @api.depends('discount')
     def _compute_amount(self):
```

Comment:
No need to keep this empty method.

Reply:
I think we need it. I had to override `_compute_amount` to add `discount` field in depends. So that change in discount field will trigger the `_compute_amount` method and will recalculate the price.
@pedrobaeza

Reply:
OK